modules/jaggery-apps/app-publisher-web/themes/appm/js/logic/flot.stat.app-endpoint.js

```diff
@@ -0,0 +1,188 @@
+var usageByContext;
+$(function () {
+    drawGraphs();
+});
+
+function drawGraphs() {
+
+    var url = window.location.pathname;
+    var comps = url.split('/');
+    var type = comps[comps.length - 2];
+    var operation = comps[comps.length - 3];
+    var action = 'getSubscriberCountByAPIs';
+    var dateRange = $('#date-range').val();
+    var from = dateRange.split('to')[0].trim() + ":00";
+    var to = dateRange.split('to')[1].trim() + ":00";
```

Comment:
Is 'dateRange' checked for mandatory prior to this level?
If not check for null below splitting